net/iproute2/BUILD

```diff
@@ -8,7 +8,8 @@ make &&
 
 prepare_install &&
 
-make SBINDIR=/usr/sbin install &&
+# this is need for ip static function
+make SBINDIR=/usr/bin install &&
```

Comment:
@Florin65 lets stop moving the binaries for now. Unless you want to update lunar-tools first., this change would leave everyone without networking (if they use static configuration).